bbn-test-4/finality-providers/README.md

```diff
@@ -88,6 +87,9 @@ previous generated.
 - `--staking-time=52560`, i.e. ~1 year
 - `--magic-bytes=62627434` `"bbt4"` as hex
 - `--covenant-committee-pks=50929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0`
+   - We use this public key to construct an unspendable script path, in order
```

Comment:
hmm this is a bit confusing, as we are already using this key to make key spending path disabled (on taproot layer) so if we would use it to disable script spending path this would mean something weird is happening.

Maybe something like:
```
Using key without known discrete logarithm as public key in some of the scripts paths, make those key paths un-spendable.  Thus the only spendable script path that remains useable is time lock path, which requires only staker key
```

wdyt ?

Reply:
Updated to a rewording of the above.